.github/actions/test-rust/action.yaml

```diff
@@ -37,7 +38,7 @@ runs:
       uses: richb-hanover/cargo@v1.1.0
       with:
         command: test
-        args: ${{ inputs.target_option }} --no-run --locked
+        args: --workspace ${{ inputs.target_option }} --no-run --locked
```

Comment:
I think this implicit with a virtual manifest, no?

Reply:
I don't think so. It says this:
```
➜  prql git:(prqlc) cargo test --exclude=prqlc
error: --exclude can only be used together with --workspace
```

Reply:
Hmm, from `cargo help test`:

> The default members of a workspace can be set explicitly with the workspace.default-members key in the root manifest. If this is not set, a virtual workspace will include all workspace members (equivalent to passing --workspace), and a non-virtual workspace will include only the root crate itself.

...though makes sense re `--exclude`; hopefully we can avoid that though